cli/templatepull.go

```diff
@@ -130,6 +131,17 @@ func (r *RootCmd) templatePull() *clibase.Cmd {
 				dest = templateName
 			}
 
+			clean, err := filepath.Abs(filepath.Clean(dest))
+			if err != nil {
+				cliui.Error(inv.Stderr, fmt.Sprintf("cleaning destination path %s failed: %q", dest, err))
+				return err
```

Comment:
Could this be an `xerrors.Errorf("lookup of absolute path failed ...: %w")` instead of the manual cliui error call? I'd expect the error to be printed twice this way.